File: cs/Iris/ExpressionElement/Instruction/ShowImageInstruction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Iris.SimpleExpressionElement;

namespace Iris.ExpressionElement.Instruction
{
    public class ShowImageInstruction : InstructionExpression
    {
        public PathExpression image;
        //public SimpleExpression position;
        public int position;

        public ShowImageInstruction()
        {

        }
    }
}

File: java/Iris/src/com/rokuan/iris/instruction/ShowImageInstruction.java
package com.rokuan.iris.instruction;

import com.rokuan.iris.enums.Constants;
import com.rokuan.iris.expression.PathExpression;

public class ShowImageInstruction extends InstructionExpression
{
	public PathExpression image;
	//public Expression position;
	public Constants.Alignment position;

	public ShowImageInstruction()
	{

	}
}
